src/Sound.cpp

```diff
@@ -0,0 +1,71 @@
+#include <iostream>
+#include "Sound.h"
+
+Sound::Sound(){
+
```

Comment:
you should make a generic Sound constructor that takes in a file and lets you play it. Right now if you want to add a lightning sound, for instance, you have to make another function